src/services/aria.ts

```diff
@@ -34,41 +37,98 @@ class Aria {
-      if (shouldDescribe) {
-        // used to ensure item is described when cursor reaches block boundaries
-        if (
-          item.parent &&
-          item.parent.ariaLabel &&
-          item.ariaLabel === 'block'
-        ) {
-          output = item.parent.ariaLabel + ' ' + itemMathspeak;
-        } else if (item.ariaLabel) {
-          output = item.ariaLabel + ' ' + itemMathspeak;
+  queue(
+    item: AriaQueueItem | AriaQueueItem[],
+    shouldDescribe: boolean = false
+  ) {
+    this.items.push(this.processQueueItems(item, shouldDescribe));
+    return this;
+  }
+
+  private processQueueItems(
+    item: AriaQueueItem | AriaQueueItem[],
+    shouldDescribe: boolean = false
+  ): string {
+    const items = Array.isArray(item) ? item : [item];
+    const processedItems = items.map((item) => {
+      if (item instanceof MQNode) {
+        // Some constructs include verbal shorthand (such as simple fractions and exponents).
+        // Since ARIA alerts relate to moving through interactive content, we don't want to use that shorthand if it exists
+        // since doing so may be ambiguous or confusing.
+        var itemMathspeak = item.mathspeak({ ignoreShorthand: true });
+        if (shouldDescribe) {
+          // used to ensure item is described when cursor reaches block boundaries
+          if (
+            item.parent &&
+            item.parent.ariaLabel &&
+            item.ariaLabel === 'block'
+          ) {
+            return item.parent.ariaLabel + ' ' + itemMathspeak;
+          } else if (item.ariaLabel) {
+            return item.ariaLabel + ' ' + itemMathspeak;
+          }
         }
+        return itemMathspeak;
       }
-      if (output === '') {
-        output = itemMathspeak;
-      }
-    } else {
-      output = item || '';
-    }
-    this.items.push(output);
-    return this;
+      return item || '';
+    });
+    return processedItems.join(' ');
   }
-  queueDirOf(dir: Direction) {
+
+  queueDirOf(
+    dir: Direction,
+    items: AriaQueueItem | AriaQueueItem[],
+    shouldDescribe: boolean = false
+  ) {
     prayDirection(dir);
-    return this.queue(dir === L ? 'before' : 'after');
+    const str = dir === L ? 'before' : 'after';
+    return this.queueDir(str, items, shouldDescribe);
   }
-  queueDirEndOf(dir: Direction) {
+
+  queueDirEndOf(
+    dir: Direction,
+    items: AriaQueueItem | AriaQueueItem[],
+    shouldDescribe: boolean = false
+  ) {
     prayDirection(dir);
-    return this.queue(dir === L ? 'beginning of' : 'end of');
+    const str = dir === L ? 'beginning of' : 'end of';
+    return this.queueDir(str, items, shouldDescribe);
+  }
+
+  private queueDir(
+    dirStr: 'before' | 'after' | 'beginning of' | 'end of',
+    items: AriaQueueItem | AriaQueueItem[],
+    shouldDescribe: boolean = false
+  ) {
+    if (
+      this.controller.ariaStringsOverrideMap &&
+      this.controller.ariaStringsOverrideMap[dirStr]
+    ) {
```

Comment:
This could be simplified to `if(this.controller.ariaStringsOverrideMap?.[dirStr])`.

```suggestion
    if (this.controller.ariaStringsOverrideMap?.[dirStr]) {
```